### PR TITLE
V3 group streaming

### PR DESCRIPTION
**Pull request by nakajima**
Relies on https://github.com/xmtp/xmtp-ios/pull/229.

### CODE REVIEW

File: Sources/XMTPiOS/Conversations.swift [hunk 1]
```diff
@@ -48,6 +63,18 @@ public actor Conversations {
 		return try await v3Client.conversations().list(opts: options).map { $0.fromFFI(client: client) }
 	}
 
+	public func streamGroups() async throws -> AsyncThrowingStream<Group, Error> {
```
Comment (nplasterer) [inline Sources/XMTPiOS/Conversations.swift:66]: I'm not sure how you're going to stream both groups and conversations at the same time. But to be able to do this in Android the stream had to return the same type.

File: Sources/XMTPiOS/Extensions/Ffi.swift [hunk 2]
```diff
@@ -193,6 +193,22 @@ extension FfiV2SubscribeRequest {
...
 
+// MARK: Messages
+
+extension FfiMessage {
+	func fromFFI(client: Client) throws -> DecodedMessage {
+		let encodedContent = try EncodedContent(serializedData: content)
+
+		return DecodedMessage(
+			client: client,
+			topic: "",
```
Comment (nplasterer) [inline Sources/XMTPiOS/Extensions/Ffi.swift:204]: If you can get the group id here that would be ideal.

**Review by nplasterer — COMMENTED** [review]
This looks good maybe a bad merge with main because of all the env stuff in the diff?
Might be worth trying to tackle the more difficult item of streaming both conversations and groups at the same time.

File: Sources/XMTPiOS/Conversations.swift (hunk 1 above)
Comment (nakajima) [reply Sources/XMTPiOS/Conversations.swift:66]: Currently planning on holding off on having a combined stream until we do the Conversation -> DM rename. Then we can just have that stream return the union.

**Pull request merged**